src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model.project;
```

Comment:
I can see that all your code works. But would it be easier if we use java `LocalDate` and `LocalDateTime` to deal with it? And it may be easier for us in the future work if we want to filter projects by deadline.